fakeredis/_server.py

```diff
@@ -645,6 +645,9 @@ def __init__(self, server):
     def shutdown(self, flags):
         self._parser.close()
 
+    def fileno(self):
+        return 0
```

Comment:
Can you add a comment about why this is needed and why the return value doesn't matter too much.